### ISSUE TITLE
Autoswitch coins Feature

### ISSUE THREAD

**Issue by codefrx** [bonesoul/hypepool#10]
It would be really great if hypepool has this feature: 

- [ autoswith coins based on profit or other user specific input.] 

This project is great.
You have all my support on this.

**Comment by bonesoul** [comment]
we can make this happen with shapeshift.io

**Comment by codefrx** [comment]
The initial idea is more like where to direct the mining hash power rather than an exchange (like shapeshift.io is).
It's more like: hash-to-coins.com
The feature should be:
You select just one stratum server, and this stratum will mine the most profitable coin at that moment autoswithing between coins.

**Comment by bonesoul** [comment]
yea that's also possible, but marking it for future.

**Comment by rst00** [comment]
You can auto switch coin only within current miner's algo. It's impossible to change algo on the fly.

**Comment by codefrx** [comment]
yes current algo, like hash-to-coins.com it will be a great feature
